base/Dockerfile

```diff
@@ -12,10 +12,14 @@ RUN conda install --yes \
     nomkl \
     numpy==1.16.2 \
     pandas==0.24.2 \
-    && conda clean -tipsy
+    && conda update conda -y \
+    && conda clean -tipsy \
+    && find /opt/conda/ -type f,l -name '*.a' -delete \
```

Comment:
It would be great to get a list of the top 10(?) large static libraries. We may decide that the conda packages of these should split out the static libraries so they don't wind up getting installed.

Reply:
openblas is the biggest one by far.

Reply:
Yep, that one I would have guessed. Am curious about the rest as there may be some things we don't know about.